src/main/java/com/bina/intervalTree/SimpleInterval1D.java

```diff
@@ -186,6 +186,23 @@ public boolean intersects(Interval1D that, double reciprocalRatio) {
     public boolean intersects(Interval1D that, double reciprocalRatio, int wiggle) {
         if (wiggle == 0) {
             return intersects(that, reciprocalRatio);
+//        } else if (this.length() == 0 && that.length() == 0) {
+          //we need to set breakend to at least 1 bp long, otherwise validation would
+            //not work as it's based on accumulated validated lengths of canonicalized
+            //variants.
+//            /*both intervals are of length 0, special case
+//            extend one interval with wiggle and see if the extended
+//            interval will cover the other interval
+//            **
+//                 **
+//
+//            ||
+//            \/
+//
+//   ---------**----------
+//                    **
+//            */
+//            return this.getLeft() + wiggle >= that.getLeft() && this.getRight() - wiggle <= that.getRight();
         }
         long len_this = reciprocalRatio == 1.0 ? this.length() : (long) Math.max(Math.ceil(this.length() * reciprocalRatio), 1l);
```

Comment:
Are you suggesting something like

```
long len_this = (long) Math.max(Math.ceil(this.length() * reciprocalRatio), 1l);
```

? I will remove length 0 interval comparison since VarSim uses length as a way to keep track of comparison results of canonicalized variants (and it's weird that two length 0 intervals overlap).